packages/mdl-ripple/index.js

```diff
@@ -1,36 +1,18 @@
-import { RippleComponent, Class, Identifier, MAX_RIPPLES } from './base';
-
-
-export default class MDLRipple {
-  static attachTo(surface) {
-   let renderer = new MDLRipple(surface);
-   let component = new RippleComponent(renderer);
-   component.isBounded = surface.getAttribute('bounded') != 'false';
-   if (surface.hasAttribute('max-radius')) {
-     component.maxRadius = parseInt(surface.getAttribute('max-radius'), 10);
-   }
-   surface.appendChild(renderer.getRootElement());
-
-   surface.addEventListener('mousedown', (ev) => {
-     let coords = RippleComponent.getNormalizedEventCoords(ev, surface);
-
-     component.startTouchBeganAnimationAtPoint(coords.top, coords.left);
-   });
-   surface.addEventListener('mouseup', (ev) => {
-     let coords = RippleComponent.getNormalizedEventCoords(ev, surface);
-
-     component.startTouchEndedAnimationAtPoint(coords.top, coords.left);
-   });
-   surface.addEventListener('mouseout', () => {
-     component.cancelAnimations();
-   });
-
-    return component;
-  }
+import MDLBaseComponent from 'mdl-base-component';
+import MDLRippleMixin, {
+  Class,
+  Identifier,
+  MAX_RIPPLES,
+  getNormalizedEventCoords,
+} from './mixin';
 
 
-  /** @param {!Element} */
-  constructor(surface) {
+export default class MDLRipple extends MDLBaseComponent {
```

Comment:
I believe currently `MDLBaseComponent` wants you to implement static getters for things like class names, string/number constants, etc. but I'm not sure if we should rethink this given our mixin architecture (cc @Garbee )